src/mmrelay/matrix_utils.py

```
@@ -1270,6 +1268,10 @@ async def connect_matrix(
     # Update the global config if a config is passed
     if passed_config is not None:
         config = passed_config
+        try:
+            config_module.relay_config = passed_config
+        except Exception as exc:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)